pymc3/distributions/continuous.py

```diff
@@ -117,6 +117,25 @@ class Uniform(Continuous):
 
        f(x \mid lower, upper) = \frac{1}{upper-lower}
 
+    .. plot::
+
+        import matplotlib.pyplot as plt
+        import numpy as np
+        x = np.linspace(-3.0, 3.0, 1000)
+        a, b = 0.0, 2.0
+        y = np.zeros(1000)
+        y[(x<b) & (x>a)] = 1.0/(b-a)
+        plt.plot(x, y, label='lower=1, upper=2')
```

Comment:
if we moved to `fig, ax = plt.subplots(); ax.plot()...` we could save a few lines with `ax.set(ylim=..., xlabel=...)`